testcases/kernel/syscalls/mmap/mmap06.c

```diff
@@ -26,10 +26,11 @@
  *  The call should fail to map the file.
  *
  * Expected Result:
- *  mmap() should fail returning -1 and errno should get set to EACCES.
+ *  mmap() should fail returning -1 and errno should get set to EBADF.
```

Comment:
Please add a note that `EACCES` is the right thing, `EBADF` is what LKL does.

Reply:
Is that not covered the the history note below?

Reply:
It's odd that it's saying this is what it should be doing.  It isn't what mmap should be returning, it's just what it temporarily is returning.